types/pusher-trigger.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface PusherTrigger {
+    readonly channel: string;
+    readonly event: string;
+    readonly payload: string;
+}
+
+export declare class PusherTriggerAPI {
+    pusherTrigger(payload: PusherTrigger): Promise<any>;

```

Comment:
I don't know what this returns but could the type be further specified rather than `any` ?